lamindb/examples/cellxgene/_cellxgene.py

```diff
@@ -45,16 +54,19 @@ def save_cxg_defaults() -> None:
     ).save()
 
     # na, unknown
-    for model, name in zip(
+    for biorecord, name in zip(
```

Comment:
I think it's more consistent to call it model, biorecord would be confusing as it sounds like it's a record, but it's the class.